### PR TITLE
Susi chat like signup verification message

### LINKED ISSUE #1810: Signup snackbar message is incomplete
**Actual Behaviour**

After Signing up the snackbar message is incomplete, it shows "You successfully signed-up! An email with a verification link was sent to your"

**Expected Behaviour**

Message should be "You successfully signed-up! An email with a verification link was sent to your address"

**Screenshots of the issue**

screenshot from 2019-01-13 21-37-43

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by sweezy793**
Fixes #1810 

Changes: Removed signup snackbar and added SUSI chat like sign up verification message

Surge Deployment Link: https://pr-1824-fossasia-susi-skill-cms.surge.sh

Screenshots for the change:

screenshot from 2019-01-15 19-46-17

### CODE REVIEW

**Comment by sweezy793** [comment]
@AakashMallik please check the changes, though for some reason reCAPTCHA is showing site owner error for production domain

**Comment by AakashMallik** [comment]
> @AakashMallik please check the changes, though for some reason reCAPTCHA is showing site owner error for production domain

Probably some key issue.. should not be a problem... @anshumanv @akshatnitd

File: src/components/Auth/SignUp/SignUp.js [hunk 1]
```diff
@@ -184,71 +184,71 @@ class SignUp extends Component {
...
-            },
-          );
+          this.setState({
+            success: false,
+            password: '',
+            loading: false,
+          });
+          openSnackBar({
+            snackBarMessage: 'Signup Failed. Try Again',
+            snackBarDuration: 6000,
```
Comment (fragm3) [inline src/components/Auth/SignUp/SignUp.js]: No need to add 6000 duration, it is set to default 4000

File: src/components/Auth/SignUp/SignUp.js [hunk 2]
```diff
@@ -184,71 +184,71 @@ class SignUp extends Component {
...
-              },
-            );
+            this.setState({
+              password: '',
+              success: false,
+              loading: false,
+            });
+            openSnackBar({
+              snackBarMessage: 'Signup Failed. Try Again',
+              snackBarDuration: 6000,
```
Comment (fragm3) [inline src/components/Auth/SignUp/SignUp.js]: No need to add 6000 duration, it is set to default 4000

Comment (sweezy793) [reply src/components/Auth/SignUp/SignUp.js]: @fragm3 Removed that! Please review the changes again

**Comment by sweezy793** [comment]
> screenshot 2019-01-16 at 3 35 38 pm
> 
> I got this error could you please check

That's a key issue, which happens on all production URLs even on SUSI's domain. If you try checking this locally, it will work :)

**Comment by fragm3** [comment]
@isuruAb captcha Key error won't be a problem

**Comment by sweezy793** [comment]
> @isuruAb captcha Key error won't be a problem. @sweezy793 squash your commits

Done. Please approve the changes :)

**Comment by sweezy793** [comment]
@akshatnitd Please verify and merge the changes

**Comment by sweezy793** [comment]
@fragm3 Please approve the changes :)

**Pull request merged**